terminology-guides/portuguese-br-terminology.md

```diff
@@ -0,0 +1,66 @@
+## Glossary For Brazilian Portuguese Translations
+
+| **Original** | **Tradução** | **Comentários** |
+| --- | --- | --- |
+| Account | Conta | |
+| Anonimity set | Conjunto de anomimato |  |
+| Blackball | Blackball (substantivo), Aplicar Blackball (quando verbo) | |
+| Blockchain | Blockchain | Mantém a palavra em inglês |
+| Block height | Altura do bloco |  |
+| Block reward | Recompensa do bloco |  |
+| Block time | Tempo do bloco |  |
+| Commit | Confirmação (substantivo), Confirmar (quando verbo) | |
+| Commitment | Comprometimento | |
+| Daemon | Daemon | Mantém a palavra em inglês |
+| Decoy | Chamariz | Termo encontrado em monero.inf.br |
+| Dust | ? | **Necessita tradução** |
+| Fork | Fork | Mantém a palavra em inglês |
+| Fungible | Fungível | |
+| Grassroots Community | ? | **Sugestão:** Comunidade de base  | 
+| Hard fork | Hard fork | Mantém a palavra em inglês |
+| Input | Entrada | Quando o termo se refere a uma transação |
+| Integrated address | Endereço integrado | |
+| Issuer | Emissor |  |
+| Key Image | Imagem chave | |
+| Ledger | Livro razão |  |
+| Ledger Wallet | Carteira Ledger |  |
+| Mine (To) | Minerar | |
+| Miner | Minerador | |
+| Mnemonic seed | Semente mnemônica | |
+| Monero Research Lab | Laboratório de Pesquisa Monero |  |
+| Multisig | Multisig | Utilizar em forma de "sigla" ou a tradução abaixo |
+| Multisignature | Assinatura múltipla | |
+| One-time Address | Endereço de uso único | |
+| Output | Saída | Quando o termo se refere a uma transação |
+| Paper wallet | Carteira em papel | |
+| Pedersen('s) Commitment | Comprometimento de Pedersen | |
+| Pool | Pool | Mantém a palavra em inglês |
+| Proof-of-work | Prova de trabalho |  |
+| Range Proof(s) | Prova de intervalo | |
+| Restore height | Altura de restauração | |
+| RingCT | RingCT | Utilizar em forma de "sigla" ou a tradução abaixo |
+| Ring confidential transaction | Transações confidenciais em anel |  |
+| Ring Member | Membro do anel |  |
+| Ring signatures | Assinaturas em anel | |
+| Ringsize | Tamanho do anel |  |
+| Recipient | Destinatário | |
+| Seed | Semente |  |
+| Sender | Remetente | |
+| Signer | Signatário | |
+| Spend Key | Chave de gasto | |
+| Stealth address | Endereços _stealth_ | Utilizar a palavra _stealth_ em itálico |
+| Subaddress | Subendereços | |
+| Sweep | Sweep (substantivo), Realizar sweep (quando verbo) | |
+| Tail emission | Emissão em cauda | |
+| Transaction unlock time | Tempo de desbloqueio da transação | |
+| Unblackball | Retirar Blackball | |

```

Comment:
Se for seguir a utilização do termo **banir**, então ficaria **desbanir** (como está na GUI).